CHANGES/9705.contrib.rst

```diff
@@ -0,0 +1 @@
+Speed up tests by disabling ``blockbuster`` fixture for ``test_static_file_huge`` and ``test_static_file_huge_cancel`` tests -- by :user:`dikos1337`.
```

Comment:
Probably should have called this `Reduced run time of tests` .. will clean that up as I'm about to do a release to fix the WebSocket regression anyways.